CISLibrary/drivers/HBridge/DRV8837.cpp

```diff
@@ -23,17 +31,60 @@ DRV8837::~DRV8837() {}
  * @param blocking will wait until operation is complete if true
  * @return mbed_error_status_t
  */
-mbed_error_status_t DRV8837::set(double value, bool blocking) {
-  return MBED_ERROR_UNSUPPORTED;
+mbed_error_status_t DRV8837::set(double value) {
+  if (value < -1.0 || value > 1.0) {
+    return MBED_ERROR_CODE_INVALID_ARGUMENT;
+  }
+
+  this->wake();
+
+  if (value == 0) {
+    return this->stop(false);
+  } else {
+    if (value < 0) {
+      this->rev = -value;
+      this->fwd = 0.0;
+    } else {
+      this->fwd = value;
+      this->rev = 0.0;
+    }
+  }
+
+  wait_us(DRV8837_SET_WAIT_TIME_US);
+  this->sleep_immediate();
+
+  return MBED_SUCCESS;
 }
 
 /**
  * @brief Stop the output, brake or coast
  *
- * @param brake will short output if true, high impedance if true
+ * @param brake will short output if true, high impedance if false
  * @param blocking will wait until operation is complete if true
  * @return mbed_error_status_t
  */
-mbed_error_status_t DRV8837::stop(bool brake, bool blocking) {
-  return MBED_ERROR_UNSUPPORTED;
+mbed_error_status_t DRV8837::stop(bool brake) {
```

Comment:
Throw in bool sleep here

Reply:
Not quite sure what you mean here. 

As in, keep the blocking argument and sleep the PS if it's true?